JS/javascript.js

```diff
@@ -0,0 +1,191 @@
+const myArray = [
+  {
+    class: 'background-img1 background-img',
+    class2: 'card-img',
+    subtitle: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.',
+    image: {
+      image1: './imgs/placeholder.png',
+      popup: './imgs/SnapshootPortfolio.png',
```

Comment:
- [ ] As pointed out by the previous code reviewers, I will however, highlight on it again and possible make is more clearer. We were asked to include a link for our images, like for example;
```
const projects = [{
    image: "link_to_image.jpg",
    link: "link_to_live_version.com",
}]
```
These attributes should be a link instead we have a directory. Currently, it is not functioning as a link because I'm unable to click on the image and link attributes